.travis.yml

```diff
@@ -95,7 +95,7 @@ install:
     - if [ -n "$PPA2" ]; then for i in `seq 0 4`; do sudo add-apt-repository "$PPA2" -y 2>&1|tee /dev/stderr|grep -q "imported" && break; sleep 30; done; fi
     - if [ -n "$DPKG_ADD_ARCH" ]; then sudo dpkg --add-architecture "$DPKG_ADD_ARCH" ; fi
     - if [ -n "$PACKAGES" ]; then travis_retry sudo apt-get update; fi
-    - if [ -n "$PACKAGES" ]; then travis_retry sudo apt-get install --no-install-recommends --no-upgrade -qq libdb4.8-dev libdb4.8++-dev $PACKAGES; fi
+    - if [ -n "$PACKAGES" ] && { [ -n "$DEP_OPTS" ] && [[ $DEP_OPTS =~ .*NO_WALLET=1.* ]]; }; then echo "Skip Berkeley DB installation"; else travis_retry sudo apt-get install --no-install-recommends --no-upgrade -qq libdb4.8-dev libdb4.8++-dev $PACKAGES; fi
```

Comment:
Hmm, maybe I understand the logic wrong here, but don't you want to split it further into the part installing `libdb*` and the part installing the rest in `$PACKAGES`?

Reply:
PACKAGES doesn't contain `libdb*` for any of the specified architectures it is just used as a trigger conditions. Basically what the `if` checked before:
- if we need to install packages from the apt subsys (i.e. PACKAGE is defined and not empty)
- then we need to install also `libdb*`

I changed it so that `libdb*` is installed only if we want to build a client without wallet functionality, i.e. `NO_WALLET=1`

Reply:
@awemany you are indeed right, I manage to miss that. Thanks for raising the issue.